Add Software as a Service Profile (#973)
spdx/spdx-3-model: model/Service/Properties/serviceHostingCountry.md

SPDX-License-Identifier: Community-Spec-1.0

# serviceHostingCountry

## Summary

Specifies a country code where a software service is hosted.

## Description

> This property should specify all countries and territories where a service is located.
> This includes indirect locations (e.g. where the data may be hosted).

Reviewer: ```suggestion
Specifies a country or territory where a software service is located.
This includes indirect locations (e.g., where the data may be hosted).
```

Update description to reflect the singular nature of the property name.
